packages/react-events/src/dom/Keyboard.js

```diff
@@ -173,12 +185,25 @@ const keyboardResponderImpl = {
     context: ReactDOMResponderContext,
     props: KeyboardProps,
   ): void {
-    const {responderTarget, type} = event;
+    const {nativeEvent, responderTarget, type} = event;
 
     if (props.disabled) {
       return;
     }
+    let defaultPrevented = (nativeEvent: any).defaultPrevented === true;
```

Comment:
We could define `const nativeEvent: any = event.nativeEvent` at the top of the function body so we don't have to do it in 3 different places in the code below.

Reply:
Addressed